core/src/apps/cardano/address.py

```diff
@@ -193,9 +238,6 @@ def derive_address_bytes(
 def _derive_byron_address(
```

Comment:
Is this function actually of any use? Let's call `derive_byron_address` directly.

Reply:
https://github.com/vacuumlabs/trezor-firmware/pull/60/commits/f9fa839bf899aa479577b3a88bf9bb34621215ee